src/ScottPlot/Plottable/RadialGaugePlot.cs

```diff
@@ -0,0 +1,709 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using ScottPlot.Drawing;
+using static System.Net.WebRequestMethods;
+
+// Credits:
+//
+// Inspired (and expanding) by https://github.com/dotnet-ad/Microcharts/blob/main/Sources/Microcharts/Charts/RadialGaugeChart.cs
+//
+// Lighten or darken color
+// https://stackoverflow.com/questions/801406/c-create-a-lighter-darker-color-based-on-a-system-color
+// https://www.pvladov.com/2012/09/make-color-lighter-or-darker.html
+// https://gist.github.com/zihotki/09fc41d52981fb6f93a81ebf20b35cd5
+//
+// Circular Segment
+// https://github.com/falahati/CircularProgressBar/blob/master/CircularProgressBar/CircularProgressBar.cs
+// https://github.com/aalitor/AltoControls/blob/on-development/AltoControls/Controls/Circular%20Progress%20Bar.cs
+//
+// http://csharphelper.com/blog/2015/02/draw-lines-with-custom-end-caps-in-c/
+//
+// Text on path
+// http://csharphelper.com/blog/2018/02/draw-text-on-a-circle-in-c/
+// http://csharphelper.com/blog/2016/01/draw-text-on-a-curve-in-c/
+
+
+// This file should be placed in this address
+// https://github.com/ScottPlot/ScottPlot/tree/master/src/ScottPlot/Plottable
+// and named RadialGaugePlot.cs
+namespace ScottPlot.Plottable
+{
+    /// <summary>
+    /// A radial gauge chart is a graphical method of displaying scalar data in the form of 
+    /// a chart made of circular gauges so that each scalar is represented by each gauge.
+    /// 
+    /// Data is managed using a single array where each element is asigned to each gauge.
+    /// Internally this data is stored in a single array and is converted to angular paramters,
+    /// through ComputeAngularData(), which are more suitable for drawing purposes and stored in a 2D array.
+    /// </summary>
+    public class RadialGaugePlot : IPlottable
+    {
+        #region Properties & fields
```

Comment:
This is a contentious topic! [Are #regions an antipattern or code smell?](https://softwareengineering.stackexchange.com/questions/53086/are-regions-an-antipattern-or-code-smell)

When you're tempted to reach for `region` it can be an indication it may be time to refactor something. Personally I only use regions in extremely large files (which itself is an anti-pattern). I find them useful to section-off large numbers of small functions, but partial classes (added in C# 2.0) are an alternative worth considering.

In this case the file isn't too big, so I'll probably remove them 👍